pkg/manager/member/scaler.go

```diff
@@ -54,38 +54,94 @@ type generalScaler struct {
 }
 
 func (gs *generalScaler) deleteDeferDeletingPVC(tc *v1alpha1.TidbCluster,
-	setName string, memberType v1alpha1.MemberType, ordinal int32) (map[int32]string, error) {
+	setName string, memberType v1alpha1.MemberType, ordinal int32) (map[string]string, error) {
 	ns := tc.GetNamespace()
 	// for unit test
-	skipReason := map[int32]string{}
+	skipReason := map[string]string{}
```

Comment:
Why change the skipReason type here, is it for the legibility？

Reply:
Now there will be multiple PVCs for one Pod, the `string` will be podName if no PVC found for the Pod and will be the pvcName if any PVC found, and then we can verify the result in the UT cases.